language/fr/email_change.lang

```diff
@@ -1,5 +1,19 @@
 <?php
 
-include("{$_EnginePath}language/en/email_change.lang");
+$_Lang['Page_Title']                            = 'Changement de l\'adresse e-mail du compte';
+
+$_Lang['Info_havetobelogged']                   = 'Tu dois être connecté pour activer une nouvelle adresse e-mail !';
+$_Lang['Info_noenoughtdata']                    = 'Le lien semble être incorrect : veuille contacter un administrateur !';
```

Comment:
Note to myself, looks like there are two missing english translations in `en/email_change.lang`.